tests/test_check.py

```diff
@@ -0,0 +1,52 @@
+from cms.models.fields import PlaceholderRelationField
+from cms.test_utils.testcases import CMSTestCase
+
+from djangocms_versioning.test_utils.factories import (
```

Comment:
Are we keeping the factories in versioning or copying them over to version locking? I started pulling them over but I don;t like repeating them as this package depends on versioning so IMO it's find to import from versioning.